src/core/meshcop/secure_transport.cpp

```diff
@@ -1035,6 +1035,9 @@ void SecureTransport::HandleMbedtlsDebug(int aLevel, const char *aFile, int aLin
         break;
 
     case 4:
+        logLevel = kLogLevelDebg;
+        break;
+
```

Comment:
Do we need to  add this? It is not really changing the code or its behavior?

The existing code already mapped `case 4` (along with `default:`) to `kLogLevelDebg` (which is set as default at the top `LogLevel logLevel = kLogLevelDebg;`.

Reply:
Oh, I missed the top default value. This is not needed.